modules/security_hub/versions.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
```

Comment:
By repo's convention, we don't enforce the required terraform and aws provider version.

Reply:
I'll remove the terraform required_version, but the aws provider version is there because there's properties in the latest version that are being used within the module.